README.md

```diff
@@ -23,15 +23,15 @@ Code Manager: Kyle Gerheiser
 
 Installers:
 
-Machine | Programmer
+Machine     | Programmer
 ------------|------------------
-Orion | Hang-Lei
-Hera | Kyle Gerheiser
-Jet | Kyle Gerheiser
-WCOSS-Dell | Hang-Lei
-cheyenne | Dom Heinzeller
-gaea | Dom Heinzeller
-WCOSS-Cray | Hang-Lei
+Hera        | Kyle Gerheiser
+Jet         | Kyle Gerheiser
+Orion       | Hang-Lei
+WCOSS-Dell  | Hang-Lei
+WCOSS-Cray  | Hang-Lei
+Cheyenne    | Dom Heinzeller
+Gaea        | Dom Heinzeller
```

Comment:
Gaea will probably change when we roll out v1.2.0

Reply:
If one leaves out `HPC-PYTHON` in the `config.sh`, it will be ignored and no python api's will be built.
I left it underfined in  Gaea, Hera and WCOSS configs.
Hera is a mess.